submitted_models/cerberus_gagarin_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,94 @@
+<!-- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+
+# CERBERUS-Aerial Scout
+This specifications.md file is a description and proof of virtual model validation for
+the CERBERUS Aerial Scout with Sensor Configuration 1. This robot may be launched using a `roslaunch` command with the variable name `CERBERUS_GAGARIN_SENSOR_CONFIG_1`.
+
+## Description
+CERBERUS Gagarin Aerial Scout is a highly autonomous system capable of being deployed in an unknown environment and proceeding for its mapping up to the point of automatic return to home. The platform basis is designed by Flyability but the rest is custom integration and new algorithms.
+In this configuration, CERBERUS Aerial Scout is equipped with the following sensors: 
+LIDAR, front facing color camera, IMU and upward facing depth sensor.
+
+## Usage Instructions
+The vehicle can be launched with the following command:
+```
+ign launch -v 4 tunnel_circuit_practice.ign worldName:=tunnel_circuit_practice_01 robotName1:=gagarin robotConfig1:=CERBERUS_GAGARIN_SENSOR_CONFIG_1 localModel:=true
+```
+The velocity of the robot can be controlled with following command:
+```
+rostopic pub /gagarin/cmd_vel geometry_msgs/Twist "linear:
+  x: -0.1
+  y: 0.1
+  z: 0.1
+angular:
+  x: 0.0
+  y: 0.0
+  z: 0.1"
+```
+
+## Usage Rights
+The software included in this package is released under a [BSD 3-Clause license](LICENSE).
+
+
+### Cost and Scale
+The CERBERUS Gagarin robot is a research platform and its cost is not publicly available at the moment.
+It weighs approximately 2.316 kg.
+
+### Sensors
+This Aerial Scout with sensor configuration 1 includes the following sensors. 
+The following specific sensors are declared payloads of this vehicle:
+* IMU - Generic, modeled by `imu` plugin
+* LIDAR - Ouster OS1-16, modeled by `gpu_ray` plugin
+* Color Camera - FLIR Blackfly S, modeled by `camera` plugin
+* Depth Sensor - Picoflexx, modeled by `gpu_ray` plugin
```

Comment:
The `gpu_ray` sensor is modeled slightly different from a `depth` camera (mainly that the distances between samples points are different). Just wondering if that's intended? The differences may be small for this sensor

Reply:
We used a `gpu_ray` plugin as I thought the` depth` sensor was used to model Kinect or Realsense - like  stereo cameras. 
Picoflexx is purely a time of flight sensor and does not capture images. Please let me know if this plugin is not correct for the model and I can change it as required.

Reply:
We have previously modeled the Picoflexx using the `depth_camera` plugin to more closely match the message types available with the [Picoflexx ROS driver](https://github.com/code-iai/pico_flexx_driver) (including depth images but not RGB). Please model using `depth_camera`. You can see examples in [this `model.sdf`](https://github.com/osrf/subt/blob/edc32fa926ff9c0ea6a413e4c046e9196909fbcb/submitted_models/marble_qav500_sensor_config_1/model.sdf#L415) and [this `vehicle_topics.launch`](https://github.com/osrf/subt/blob/edc32fa926ff9c0ea6a413e4c046e9196909fbcb/submitted_models/marble_qav500_sensor_config_1/launch/vehicle_topics.launch#L80).

Reply:
Thanks @iche033  @angelacmaio , I have modelled picoflexx using `depth_camera` plugin.